server/apps/publish/archive_publish.py

```diff
@@ -718,3 +718,9 @@ def on_update(self, updates, original):
     include_states=['published', 'corrected'],
     privileges=['correct']
 )
+
+superdesk.workflow_action(
```

Comment:
Should be part of `archive/__init__.py` module so that everything related to rewrite will be in one place. I know correction and kill are in `archive/__init__.py` but @mdhaman refactored them.